src/index.ts

```diff
@@ -45,6 +45,14 @@ class SMTPConnector extends BaseConnector<SMTPConnectorOptions, null> {
 
   async send(message: EmailMessage): Promise<SentMessageInfo | undefined> {
     if (!this.transporter) return
+    if (typeof message.to !== 'string' && !Array.isArray(message.to)) {
```

Comment:
Do we need to check if the array | string is empty?

Reply:
I am not sure. I am also wanting to add cc, bcc later - so might do it